superflore/generators/bitbake/yocto_recipe.py

```diff
@@ -173,5 +174,17 @@ def get_recipe_text(self, distributor, license_text):
         ret += 'SRC_URI[sha256sum] = "' + self.src_sha256 + '"\n'
         ret += 'S = "${WORKDIR}/'
         ret += self.get_src_location() + '"\n\n'
+        # Check for patches
+        if self.patch_files:
+            ret += 'SRC_URI += "\\\n'
+            ret += ' \\\n'.join(self.patch_files) + '"\n\n'
+        if self.name == 'catkin':
```

Comment:
Can these be integrated into patches?

Reply:
In the normal sense, no. I think I could, however, just detect an existing `.inc` file and include it.